lib/modules/manager/tflint-plugin/index.ts

```diff
@@ -0,0 +1,12 @@
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+
+export { extractPackageFile } from './extract';
+
+// Only from GitHub Releases: https://github.com/terraform-linters/tflint/blob/master/docs/developer-guide/plugins.md#4-creating-a-github-release
+export const supportedDatasources = [GithubReleasesDatasource.id];
+
+export const defaultConfig = {
+  commitMessageTopic: 'TFLint plugin {{depName}}',
+  fileMatch: ['\\.tflint\\.hcl$'],
+  extractVersion: '^v(?<version>.*)$',
```

Comment:
Do you expect that every release MUST have a `v` prefix? Please double check that TFLint also fails if no `v` prefix is present. Otherwise we should make the `v` optional

Reply:
TFlint's configuration file expects to be configured with the version without the `v` prefix, but TFLint seems to enforce it when fetching the plugins when it initializes.

For instance:

* [TFLint's function to fetch the release from GitHub](https://github.com/terraform-linters/tflint/blob/06c3ef71762ed6799490c67cdf21bf1673ff4a8d/plugin/install.go#L154-L155)
* The [`TagName` function enforces the release to start with a `v`](https://github.com/terraform-linters/tflint/blob/06c3ef71762ed6799490c67cdf21bf1673ff4a8d/plugin/install.go#L45-L50).

It's also documented in several places:

* [The `Install` function that downloads releases from GitHub](https://github.com/terraform-linters/tflint/blob/06c3ef71762ed6799490c67cdf21bf1673ff4a8d/plugin/install.go#L58-L74)
* [The developer's documentation](https://github.com/terraform-linters/tflint/blob/master/docs/developer-guide/plugins.md#4-creating-a-github-release)

I would expect releases not starting with a `v` prefix to not be discoverable by TFLint, so I would keep the the regex as it is.

Reply:
Great, thanks for looking into that.